src/Components/UsersList.js

```diff
@@ -19,14 +19,14 @@ const TableBodyCellStyled = styled(TableCell)`
 `;
 
 const TableContainerStyled = styled(TableContainer)`
-    margin-top:200;
-    margin-left:200;
-    margin-right:100;
+    margin-top:200px;
+    margin-left:200px;
+    margin-right:100px;
```

Comment:
It's better that the space from top is  100 pixels, `margin-top:100px`, and space from right is 200 pixels, `margin-right: 200px`.